packages/dx-react-grid/docs/guides/plugin-overview.md

```diff
@@ -11,19 +27,9 @@ Note that the plugins are composable and can be nested into each other.
 
 Refer to the [Reference](../reference/grid.md) to see the complete plugin list.
 
-## Plugin Order
-
-The plugin order is important. Plugins implementing an interface should be linked before the plugin that uses it. For example, a data processing plugin is based on some state, and should follow the appropriate state plugin. Some visualization plugins extend the `Table`'s functionality and should follow it in the code. This rule is demonstrated in the following example:
-
-.embedded-demo(grid-filtering/filter-row)
-
-NOTE: Refer to the plugin's reference for information on its dependencies.
-
-The data processing plugins' order is also important because they transform data in the same order they appear. For example, if the `IntegratedPaging` plugin precedes the `IntegratedFiltering`, the Grid filters the current page's data. Swap the plugins to paginate filtered data.
-
 ## UI Plugins
 
-The Grid's UI plugins use templates to render the UI. A template is a function that returns a [ReactElement](https://facebook.github.io/react/docs/react-api.html#createelement) depending on the argument values. You can implement your templates or use one of the predefined template suites:
+The Grid's UI plugins use special components to render the UI. You can implement your own suite of components or chose between the predefined ones:
```

Comment:
> You can implement your own suite of components or chose between the predefined ones

I'd change the priority... something like this:

You can use one of the predefined component suites or implement your own one.

Reply:
Did not think about that, simply updated the existing sentence. @viterobk, what do you think at this point?

Reply:
Currently, the sentence brings a reader to the components list that follows the sentence. Transposition of the "choose..." and "implement..." parts, will require the additional connection between the sentence and the list, like "The following predefined components are available". For me, it's better to leave this paragraph as is.